css/selectors4/focus-within-001.html

```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```

Comment:
Ok, I've done this in all the tests.

I'm still removing the empty `flags`, but I can add it if it's a blocker to land this. I have seen many tests that don't have it.